hbase-server/src/main/java/org/apache/hadoop/hbase/master/MasterCoprocessorHost.java

```diff
@@ -297,6 +297,24 @@ public void call(MasterObserver observer) throws IOException {
     });
   }
 
+  public void preListNamespaces(final List<String> namespaces) throws IOException {
+    execOperation(coprocEnvironments.isEmpty() ? null : new MasterObserverOperation() {
+      @Override
+      public void call(MasterObserver oserver) throws IOException {
```

Comment:
Just a minor spelling nit: 
```suggestion
      public void call(MasterObserver observer) throws IOException {
```